### DISCUSSION TITLE
Update api dynamically

### DISCUSSION THREAD

**Discussion by spyshow** [strapi/strapi#6017] [Legacy Topics]
hi 

i have an application that the server create tables in the database when admin add new machines,
do strapi have a "watch" function for the database , so the api changes ( add new endpoint ) for the new added machines ??

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by lauriejim** [comment]
Hello! You are creating new tables in your database and you want Strapi creates a new Content Type in your application?

**Reply by spyshow** [reply]
yes so i can make rest api calling on it

**Reply by lauriejim** [reply]
This feature doesn't exist in Strapi.

**Comment by lauriejim** [comment]
Strapi doesn't provide feature that watch your database structure and update it's Content Type structure automatically.

And there is no really workaround for that now.
